downloader/src/main/kotlin/Downloader.kt

```diff
@@ -76,16 +102,10 @@ class Downloader {
     }
 
     /**
-     * Download the source code of the [target] package to a folder inside [outputDirectory]. The folder name is created
-     * from the [name][Identifier.name] and [version][Identifier.version] of the [target] package [id][Package.id].
-     *
-     * @param target The description of the package to download.
-     * @param outputDirectory The parent directory to download the source code to.
-     * @param allowMovingRevisions Indicate whether VCS downloads may use symbolic names to moving revisions.
-     *
-     * @return The [DownloadResult].
-     *
-     * @throws DownloadException In case the download failed.
+     * Download the source code of the [target] package to a sub-directory inside [outputDirectory]. The sub-directory
+     * hierarchy is inferred from the [name][Identifier.name] and [version][Identifier.version] of the package. The
+     * [allowMovingRevisions] parameter indicates whether VCS downloads may use symbolic names for moving revisions. A
```

Comment:
I think "moving revision" is no established term, maybe explain it by adding something along the lines of "for example a Git branch, compared to a Git tag which is supposed to be stable".

Reply:
I left that part of the sentence unmodified compared to before, but I've reworded it now as part of these improvements.